src/gui/transferlistmodel.cpp

```diff
@@ -268,6 +429,55 @@ QVariant TransferListModel::data(const QModelIndex &index, const int role) const
     return {};
 }
 
+QVariant TransferListModel::data(const QModelIndex &index, const int role) const
+{
+    if (!index.isValid()) return {};
+
+    const BitTorrent::TorrentHandle *torrent = m_torrentList.value(index.row());
+    if (!torrent) return {};
+
+    switch (role) {
+    case Qt::ForegroundRole:
+        return stateForeground(torrent->state());
+    case Qt::DisplayRole:
+        return displayValue(torrent, index.column());
+    case UnderlyingDataRole:
+        return internalValue(torrent, index.column());
+    case AdditionalUnderlyingDataRole:
+        return internalValue(torrent, index.column(), true);
+    case Qt::DecorationRole:
+        if (index.column() == TR_NAME)
+            return getIconByState(torrent->state());
+        break;
+    case Qt::TextAlignmentRole:
+        switch (index.column()) {
+        case TR_AMOUNT_DOWNLOADED:
+        case TR_AMOUNT_UPLOADED:
+        case TR_AMOUNT_DOWNLOADED_SESSION:
+        case TR_AMOUNT_UPLOADED_SESSION:
+        case TR_AMOUNT_LEFT:
+        case TR_COMPLETED:
+        case TR_SIZE:
+        case TR_TOTAL_SIZE:
+        case TR_ETA:
+        case TR_SEEDS:
+        case TR_PEERS:
+        case TR_UPSPEED:
+        case TR_DLSPEED:
+        case TR_UPLIMIT:
+        case TR_DLLIMIT:
+        case TR_RATIO_LIMIT:
+        case TR_RATIO:
+        case TR_QUEUE_POSITION:
+        case TR_LAST_ACTIVITY:
+        case TR_AVAILABILITY:
+            return QVariant {Qt::AlignRight | Qt::AlignVCenter};
```

Comment:
Shouldn't we consider right-to-left locales? (aka reverse this?)

Reply:
I found this: `Qt::AlignAbsolute` in https://doc.qt.io/qt-5/qt.html so maybe it is already handled by Qt? Warrants a test to find out.

Reply:
I wouldn't want to touch it in this PR.

Reply:
ok